cacheops/query.py

```diff
@@ -261,6 +265,8 @@ def iterator(self):
                 if cache_data is not None:
                     results = pickle.loads(cache_data)
                     for obj in results:
+                        # Notify about cache hit
+                        self._send_post_lookup_signal(hit_cache=True)
```

Comment:
Oops. I will fix this and add a test that ensures signal will be emitted only once per queryset evaluation